RUST/nstream-iter/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "nstream"
+version = "0.1.0"
+authors = ["Jeff Hammond <jeff.r.hammond@intel.com>"]
```

Comment:
Do you want to put your name here?

Reply:
Can do.
Wasn't sure what the best approach is for files where I only changed a couple of lines (applies to the copyright too).

Reply:
It's a BSD research project with no downstream productization so I don't think it matters but I encourage everyone to be acknowledged by default.